vaccine/tests/test_vaccine_reg_ussd.py

```diff
@@ -1139,10 +1142,10 @@ async def test_state_success(evds_mock):
         "information and appointment details will be sent via SMS."
     )
     assert reply.session_event == Message.SESSION_EVENT.CLOSE
-    assert u.state.name == "state_age_gate"
```

Comment:
We should still end up in `state_age_gate` after we've submitted to EVDS

Reply:
yup, that's correct. I thought it was redundant since we actually have a session close event. I've re-added this assert